### PR TITLE
fix bug: sealos run automatically obtain system architecture information

### PR DESCRIPTION
**Pull request by lixiaojun2914**
sealos run automatically obtain system architecture information

### CODE REVIEW

**Comment by cuisongliu** [comment]
There may be something wrong with this modification. You need to execute the arch in each node to get the arch corresponding node to assign values.

**Comment by fanux** [comment]
git commit -s --amend && git push

need signoff your PR

File: pkg/apply/run.go [hunk 1]
```diff
@@ -157,11 +158,48 @@ func (r *ClusterArgs) SetClusterRunArgs(imageList []string, args *RunArgs) error
 			masters := stringsutil.SplitRemoveEmpty(args.Cluster.Masters, ",")
 			nodes := stringsutil.SplitRemoveEmpty(args.Cluster.Nodes, ",")
 			r.hosts = []v2.Host{}
-			if len(masters) != 0 {
-				r.setHostWithIpsPort(masters, []string{v2.MASTER, string(v2.AMD64)})
+
+			clusterSSH := r.cluster.GetSSH()
+			ssh := ssh.SSH{
```
Comment (fanux) [inline pkg/apply/run.go:163]: Set ssh info before new the ssh client

File: pkg/apply/run.go [hunk 2]
```diff
@@ -157,11 +158,48 @@ func (r *ClusterArgs) SetClusterRunArgs(imageList []string, args *RunArgs) error
...
+				arch := string(cmd)
+				if arch == "x86_64" {
+					amd64MasterList = append(amd64MasterList, master)
+				} else if arch == "aarch64" {
+					arm64MasterList = append(arm64MasterList, master)
+				}
+			}
+			// nodes grouped by architecture
+			for _, node := range nodes {
+				cmd, _ := ssh.Cmd(node, "arch")
```
Comment (fanux) [inline pkg/apply/run.go:184]: redundant code

File: pkg/apply/run.go [hunk 3]
```diff
@@ -157,11 +158,48 @@ func (r *ClusterArgs) SetClusterRunArgs(imageList []string, args *RunArgs) error
 			masters := stringsutil.SplitRemoveEmpty(args.Cluster.Masters, ",")
```
Comment (fanux) [inline pkg/apply/run.go:158]: return fast, if !r.cluster.CreationTimestamp.IsZero() {return nil}

**Comment by cuisongliu** [comment]
need support apply scale ,when  change node and master support it

**Comment by fanux** [comment]
@lixiaojun2914 you need to signoff this PR

git commit -s --amend && git push

need signoff your PR

**Pull request closed without merging**